src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesControl.xaml.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Input;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    /// <summary>
+    /// Interaction logic for ConfigurationFilesControl.xaml
+    /// </summary>
+    public partial class ConfigurationFilesControl : UserControl
+    {
+        private ConfigurationFilesViewModel _viewModel;
+
+        public ConfigurationFilesControl(ConfigurationFilesViewModel configurationFiles)
+        {
+            InitializeComponent();
+            _viewModel = configurationFiles;
+            DataContext = configurationFiles;
+        }
+
+        private void ListViewItem_MouseDoubleClick(object sender, MouseButtonEventArgs e)
+        {
+            _openButton.Command.Execute(null);
+        }
+
+        internal void InitializeOnActivated()
+        {
+            _viewModel.SetConfigPaths();
+        }
+
+        private void Hyperlink_RequestNavigate(object sender, System.Windows.Navigation.RequestNavigateEventArgs e)
+        {
+            if (sender is Hyperlink)
```

Comment:
We shouldn't have to check if the sender is a hyperlink since the method signature of this method requires that a RequestNavigateEventArgs be passed as the second argument and that's really what we care about. From the RequestNavigateEventArgs we want the Uri. In my comment I left previously I wrote what the code for this method should look like.